crates/router/src/core/payment_methods/cards.rs

```diff
@@ -2435,10 +2435,17 @@ pub async fn list_payment_methods(
     )
     .await?;
 
+    let profile_id = profile_id
```

Comment:
business profile can be used here too?

Reply:
@Aprabhat19 can you check whether this api is being used with api key anywhere? Because we will not have context of business profile when used with api key and it would fail at this point

Reply:
Its not being used with api -key